momo_modules/multi_sensor_module/src/pulse.c

```diff
@@ -1,46 +1,55 @@
 #include "pulse.h"
-#include "timer1.h"
+#include "timerx.h"
 #include "sample.h"
-#include "sensor_defines.h"
 #include "port.h"
+#include "ioc.h"
+#include "ccp.h"
 #include <xc.h>
 
 
 uint16_t counter;
+uint16_t periods;
```

Comment:
I don't think this is ever explicitly initialized to 0, as `counter` is in the module's initialization() function.